packaging/docker/docker-compose/.env

```diff
@@ -0,0 +1,20 @@
+
+# ALFRESCO REPOSITORY
+REPOSITORY_IMAGE=quay.io/alfresco/alfresco-full-content-services
```

Comment:
Please check with the Repository Team (@andrei-rebegea) on the correct image to use.

Reply:
I talked with Andrei and we are going to use the alfresco-full-content-services image until the new image for the repo will be stable. I will open a task to update the repository link when the new created image will be stable.